packages/patternfly-4/react-integration/cypress/integration/backgroundimage.spec.ts

```diff
@@ -0,0 +1,11 @@
+describe('Background Image Demo Test', () => {
+  it('Navigate to demo section', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('#backgroundimage-demo-nav-item-link').click(); 
+    cy.url().should('eq', 'http://localhost:3000/backgroundimage-demo-nav-link');
+  }); 
+
+  it('Verify filter has correct class', () => {
+    cy.get('svg').should('have.class', 'pf-c-background-image__filter')
```

Comment:
Should we have a test if there's no svg filter?

Reply:
I made the svg filter optional for this, but I'm not sure what kind of test to run for that in the demo since I can only load one background image. Thoughts?